app/src/main/java/Activities/ViewProblemActivity.java

```diff
@@ -63,7 +65,7 @@ protected void onCreate(Bundle savedInstanceState) {
         //Extract selected problem object through intent and index of problem list
         Intent intent = getIntent();
         this.position = intent.getIntExtra("position",0);
-        this.userId = intent.getStringExtra("USERIDEXTRA");
+        this.userId = intent.getStringExtra(USERIDEXTRA); // asdasdasdasdasdasdASDASDASDAS
```

Comment:
what is // asdasdasdasdasdasdASDASDASDAS?

Reply:
sorry that was just a marker i forgot to remove it